packages/forms/examples/tiles.md

```diff
@@ -0,0 +1,91 @@
+The `Tiles` component is implemented using the [W3C Radio Group pattern](https://www.w3.org/TR/wai-aria-practices/#radiobutton).
+Each `Tile` may be treated as an individual radio input.
+
+```jsx
+const { Well } = require('@zendeskgarden/react-notifications/src');
+const ChevronIcon = require('@zendeskgarden/svg-icons/src/16/chevron-box-stroke.svg').default;
+const CheckboxDoubleIcon = require('@zendeskgarden/svg-icons/src/16/check-box-double-stroke.svg')
+  .default;
+const TextIcon = require('@zendeskgarden/svg-icons/src/16/text-stroke.svg').default;
+const MultilineIcon = require('@zendeskgarden/svg-icons/src/16/multiline-stroke.svg').default;
+const CheckIcon = require('@zendeskgarden/svg-icons/src/16/check-lg-stroke.svg').default;
+const NumberIcon = require('@zendeskgarden/svg-icons/src/16/number-stroke.svg').default;
+const DecimalIcon = require('@zendeskgarden/svg-icons/src/16/decimal-stroke.svg').default;
+const CalendarIcon = require('@zendeskgarden/svg-icons/src/16/calendar-stroke.svg').default;
+const CreditCardIcon = require('@zendeskgarden/svg-icons/src/16/credit-card-stroke.svg').default;
+const AsterickIcon = require('@zendeskgarden/svg-icons/src/16/asterisk-stroke.svg').default;
```

Comment:
* `AsteriskIcon` :wink: #rickroll

Reply:
Yes, I’d initially designed this to be used for checkboxes too. We don’t currently have a use case, but from a design perspective we’re approved and prepared to introduce that concept whenever the engineering effort makes sense.

Reply:
Updated.